src/main/java/utils/PropertyReader.java

```diff
@@ -0,0 +1,27 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+
+    public static String read(String property) {
+
+        try (InputStream input = new FileInputStream("src/test/resources/properties/hw3_test_data.properties")) {
```

Comment:
I recommend to pass path to the file as parameter

Reply:
Done